src/stream/src/executor_v2/lookup/tests.rs

```diff
@@ -239,11 +240,14 @@ async fn test_lookup_this_epoch() {
 
     for (k, v) in store.scan::<_, Vec<u8>>(.., None, u64::MAX).await.unwrap() {
         let mut deserializer = memcomparable::Deserializer::new(v);
-        println!(
-            "{:?} => {:?}",
-            k,
-            deserialize_datum_from(&DataType::Int64, &mut deserializer).unwrap()
-        );
+        // Do not deserialize datum for SENTINEL_CELL_ID cuz the value length is 0.
+        if deserialize_column_id(&k[k.len() - 4..]).unwrap() != SENTINEL_CELL_ID {
```

Comment:
I'm not sure the optimization is worth. We might need this check every time it's possible to get a sentinel?

Reply:
In the future, we can use relational table API to get data. All tests should be refactored to use relational table API instead of getting a single k-v pairs.

Here I think a better way is to check if `v` is empty. Also should use `deserialize_cell` instead of `deserialize_datum`.